lib/playground.dart

```diff
@@ -196,7 +196,18 @@ class Playground implements GistContainer, GistController {
       blankGist.getFile('index.html').content = url.queryParameters['html'];
       blankGist.getFile('styles.css').content = url.queryParameters['css'];
       editableGist.setBackingGist(blankGist);
-    } else if (_gistStorage.hasStoredGist && _gistStorage.storedId == null) {
+    } 
+    else if (url.hasQuery &&
+        url.queryParameters['export'] != null) {
```

Comment:
We had talked about using the `id` param like regular github gists, just running some heuristics on the id to see if it was a special export id. Not ciritical, but if we don't do it in this PR, we should add a TODO: to explore it later.